File: ColorSoft.Web/Data/Models/Message.cs
using System;

namespace ColorSoft.Web.Data.Models
{
    public class Message
    {
        public Guid Id { get; set; }
        public string Subject { get; set; }
        public string MessageText { get; set; }
        public string From { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: ColorSoft.Web/Scripts/applications/colorsoft/models/message.js
angular.module("colorSoft").factory("Message", ['ModelBase', function(ModelBase) {
    return function(args) {

        args = args || {
            Id: '',
            Subject: '',
            Message: '',
            CreatedAt: '',
            Email: ''
        };

        angular.extend(this, args, new ModelBase());
    };
}]);